redbot/core/dev_commands.py

```diff
@@ -115,7 +115,26 @@ def sanitize_output(ctx: commands.Context, input_: str) -> str:
 
         result = self.sanitize_output(ctx, str(result))
 
-        await ctx.send(box(result, lang="py"))
+        def check(m):
+            return m.content.lower() == "more"
+
+        pages = list(pagify(result, shorten_by=18))
+        for idx, page in enumerate(pages, 1):
+            if idx != 1:
+                last = await ctx.send(_("There are still {} messages. "
```

Comment:
Can you add this "more" functionality as a method in utils? I have a feeling it will come in handy for a lot of things.

Reply:
I agree, could be expanded on in the future too, reaction buttons would work quite nicely. But another PR for that.

Reply:
I agree, could be expanded on in the future too as a reaction-based menu, but another PR.